CHANGELOG.md

```diff
@@ -36,6 +36,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * added `btlx_processes` folder and processes `JackCut` and `FrenchRidgeHalfLap`
 * added `BTLx` Grasshopper component
 * added `FrenchRidgeHalfLap` joint
+* added `L-HalfLapJoint`
+* added `T-HalfLapJoint`
+* added `ShowTopologyTypes` GH Component
```

Comment:
these are in the wrong place, they should be above under the unreleased changes section.